### ISSUE TITLE
Handle the upcoming Atom Electron update

### ISSUE THREAD

**Issue by Arcanemagus** [abe33/atom-pigments#304]
It sounds like this may have already been discussed, but just in case it gets forgotten about I'm filing this issue to track the handling of the editor rendering rewrite done in https://github.com/atom/atom/pull/13880. Currently this is causing the Pigments package to crash on launch with a `Failed to activate the pigments package, cannot find highlights-component in the require cache` message.

When https://github.com/atom/atom/pull/12696 gets merged and released as a beta this will start causing problems (as it does currently for anyone testing the branch 😉).

**Comment by clijiac** [comment]
Atom 1.19.beta0 is out, seems have this issues you said

**Comment by bjmiller** [comment]
Here's a stack trace, to help out.

**Atom**: 1.19.0-beta0 x64
**Electron**: 1.6.9
**OS**: Mac OS X 10.11.6
**Thrown From**: [pigments](https://github.com/abe33/atom-pigments) package 0.39.1


### Stack Trace

Failed to activate the pigments package

```
At Cannot find 'highlights-component' in the require cache.

Error: Cannot find 'highlights-component' in the require cache.
    at requireCore (/packages/pigments/lib/pigments.coffee:310:19)
    at Object.patchAtom (/packages/pigments/lib/pigments.coffee:312:26)
    at Object.activate (/packages/pigments/lib/pigments.coffee:16:6)
    at Package.module.exports.Package.activateNow (/Applications/Atom Beta.app/Contents/Resources/app/src/package.js:253:25)
    at /Applications/Atom Beta.app/Contents/Resources/app/src/package.js:225:38
    at Package.module.exports.Package.measure (/Applications/Atom Beta.app/Contents/Resources/app/src/package.js:99:21)
    at /Applications/Atom Beta.app/Contents/Resources/app/src/package.js:218:32
    at Package.module.exports.Package.activate (/Applications/Atom Beta.app/Contents/Resources/app/src/package.js:215:40)
    at PackageManager.module.exports.PackageManager.activatePackage (/Applications/Atom Beta.app/Contents/Resources/app/src/package-manager.js:645:40)
    at /Applications/Atom Beta.app/Contents/Resources/app/src/package-manager.js:626:35
    at Config.module.exports.Config.transactAsync (/Applications/Atom Beta.app/Contents/Resources/app/src/config.js:346:24)
    at PackageManager.module.exports.PackageManager.activatePackages (/Applications/Atom Beta.app/Contents/Resources/app/src/package-manager.js:621:25)
    at PackageManager.module.exports.PackageManager.activate (/Applications/Atom Beta.app/Contents/Resources/app/src/package-manager.js:603:52)
    at /Applications/Atom Beta.app/Contents/Resources/app/src/atom-environment.js:843:36
```

**Comment by Greenek** [comment]
Fixed by #319
